### PR TITLE
Expense page

### PR DESCRIPTION
**Pull request by FNKMYR**
Rather large user story, thus thanks in advance for reviewing.

### CODE REVIEW

File: src/components/ExpenseList.js [hunk 1]
```diff
@@ -4,23 +4,35 @@ import Expense from './Expense.js';
...
     ]);
-    setInputValue('');
+    setInputValue({ title: '', amount: '' });
+    console.log(inputValue.amount.type);
+    console.log(totalExpenses.type);
   };
+  const totalExpenses = Number(
+    expenses
+      .map(expense => expense.amount)
+      .reduce((prev, curr) => +prev + +curr, 0)
```
Comment (teifler) [inline src/components/ExpenseList.js:8]: Not sure if you need the double ++ in line 27

**Review by Quaquaro — APPROVED** [review]
Nice clean code. Just one more thing, the variable p that you use to format your numbers, you could name it better.
I'm very curious to see what your app looks like in terms of styling. Lasse has spied that you have probably built an amazing prototype for Figma.

File: src/components/ExpenseList.js (hunk 1 above)
Comment (FNKMYR) [reply src/components/ExpenseList.js:8]: Necessary, otherwise the values are read as strings. Tried it without.

**Comment by FNKMYR** [comment]
> Looks good! Just one question regarding autocomplete? You have to put autocomplete in the form and the inputs to make it work?

Still baffles me too, but that's what seemed to work. Tried a few variants.

**Pull request merged**